src/main/java/com/smalaca/messagesender/controller/ErrorHandler.java

```diff
@@ -0,0 +1,18 @@
+//package com.smalaca.messagesender.controller;
+//
+//import org.springframework.boot.autoconfigure.web.ErrorController;
+//import org.springframework.web.bind.annotation.RequestMapping;
+//import org.springframework.web.bind.annotation.RestController;
+//
+//@RestController
+//public class ErrorHandler implements ErrorController {
+//    @RequestMapping("/error")
```

Comment:
![image](https://user-images.githubusercontent.com/4753177/36960907-4bf28d2c-2049-11e8-98f2-5a7018b9c9b0.png)

Jak coś jest wykomentowane to należy to usunąć